scripts/documentation/internal-link-checker.ts

```diff
@@ -83,38 +83,6 @@ function readSiteMapLinks(filePath: string): string[] {
   return sitemap.urlset.url.map((obj) => obj.loc);
 }
 
-/**
- * This function checks if a link is for a private package.
- * When link is for a private package, it is not included in the sitemap.
- * However, some shared docs might be written for this private package during development.
- * @param link e.g. /nx-api/vite/generators/configuration
- * @returns true if the link is for a private package or NODE_ENV is not development, false otherwise.
- */
-function checkLinkIsForPrivatePackage(link: string) {
```

Comment:
This function doesn't actually help.

1. Documentation inside a private package is not checked for broken links because it doesn't get generated into the `/doc` folder.
2. Links from normal docs to docs for a private package should throw an error, but this check was silencing that warning for some reason.